docs/ProductImages.md

```diff
@@ -51,6 +51,8 @@ Specification:
 | `aspectRatio`             | `string`                                   | Sets the aspect ratio of the image; that is, whether the image should be square, portrait, landscape, etc. The value should follow the [common aspect ratio notation](https://en.wikipedia.org/wiki/Aspect_ratio_(image)), i.e., two numbers separated by a colon (e.g, "1:1" for square, "3:4" for upright portrait, or even large values such as "1920:1080") | `"auto"`          |
 | `showNavigationArrows`             | `boolean`                                   | Controls if the navigation arrows should appear | `true`          |
 | `showPaginationDots`             | `boolean`                                   | Controls if the pagination dots should appear | `true`          |
+| `thumbnailAspectRatio`             | `string`                                   | Sets the aspect ratio of the thumbnail image; that is, whether the image should be square, portrait, landscape, etc. The value should follow the [common aspect ratio notation](https://en.wikipedia.org/wiki/Aspect_ratio_(image)), i.e., two numbers separated by a colon (e.g, "1:1" for square, "3:4" for upright portrait, or even large values such as "1920:1080") | `"auto"`          |
```

Comment:
I think you should make this shorter because of the explanation of aspect ratio is just above it